### ISSUE TITLE
DisplayUsers

### ISSUE THREAD

**Issue by suryanevcodes** [suryanevcodes/healthfyme-#3]
Personal Details Management: Develop functionalities for users to input and manage personal details such as weight, height, and age. Ensure data validation and integrity.

**Comment by suryanevcodes** [comment]
moula try to complete it and review it within today

**Comment by MoulaNeverCodes** [comment]
Yeah, the code looks good but its not relevant to the given task. please go through the code and make the required changes. @suryanevcodes

**Comment by MoulaNeverCodes** [comment]
completed and verified push it to the main branch and don't do any updates now
